### ISSUE TITLE
Do not use hard coded id for document-types

### ISSUE THREAD

**Issue by lukadschaak** [teamneusta/pimcore-presentation-bundle#7]
In the Installer the `document-types.php` is used to install document types of the bundle.

Maybe it is better to let Pimcore choose an id instead of using a hard coded ID.

**Comment by lukadschaak** [comment]
@storbahn We talked about the IDs yesterday. Do you think it is possible to remove them from the `document-types.php` file and just iterate over all document types when checking if the bundle is installed?

**Comment by jdreesen** [comment]
The problem is that the installation/uninstallation uses the IDs. 

But as Pimcore seems to use a UUIDv4 as default (if it isn't set) it would probably be fine to use any (vendor-prefixed) string as ID.

**Comment by lukadschaak** [comment]
We thought that instaed of using the IDs, we could iterate over all document types and check other attributes like the name or smiliar.

**Comment by jdreesen** [comment]
Yes, that'll probably work. But using IDs would be easier, I think. And maybe it's nicer to use explicit IDs instead of UUIDs for users who don't store them in the database because they are easier to recognize.
